src/vim-loader/bfast.ts

```diff
@@ -1,4 +1,6 @@
-class BFastHeader {
+import { Logger } from './logger'
```

Comment:
While the Logging functions were a useful tool for debugging during development, but I don't think it provides the user of this API much value, and it will complicate separating BFAST out into its own package. Just food for thought, we can leave it in now of course.